erpnext/manufacturing/doctype/routing/routing.py

```diff
@@ -11,6 +11,15 @@
 class Routing(Document):
 	def validate(self):
 		self.set_routing_id()
+		self.update_op_time()
+
+	def update_op_time(self):
+		bom_list = frappe.db.sql("""select name, routing from `tabBOM`
+			where routing = %s""", (self.name))
+		for bom_no in bom_list:
+			frappe.db.sql("""update `tabBOM Operation` set time_in_mins = %s
+				where parent = %s """,
+				(self.operations[0].time_in_mins, bom_no[0]))
```

Comment:
I am not understanding this part well. All operations are getting set to the same value? `self.operations[0].time_in_mins`

Reply:
Updated it, previous one was incorrect.